content/blog/new-powershell-library-for-the-hpe-greenlake-cloud-platform.md

```diff
@@ -0,0 +1,405 @@
+---
+title: New PowerShell library for the HPE GreenLake platform
+date: 2023-04-26T11:33:07.486Z
+externalLink: ""
+author: Lionel Jullien
+authorimage: /img/small-size-id.jpg
+disable: false
+tags:
+  - GreenLake
+  - GLCP
+  - PowerShell
+  - Common cloud services
```

Comment:
we don't need this tag, even if it's close to the user border.
Wondering if "GreenLake edge-to-cloud platform" but it's a bit long

Reply:
According to Denis I must use the already in place tag: hpe-greenlake-cloud-platform